### ISSUE TITLE
steam install error

### ISSUE THREAD

**Issue by DadyRick** [ChrisTitusTech/winutil#3001]
as of using the win utils on my fresh install of win10 pro i got a error for the install of steam from chocolatey. ill leave a screen shot to show the error hope it helps as I'm loving this project.

![image](https://github.com/user-attachments/assets/5520748f-a0c2-43c9-88d0-88cc196ce961)

**Comment by Marterich** [comment]
TLDR:
This seems like a Chocolatey error and there is nothing we really can do about it. 

Background:
When the application link server is added to Chocolatey, a hash of the installer is created to ensure it doesn't change or get replaced by a malicious version. If (like I suppose happened here) Steam replaces the download file with an updated version, but the Chocolatey package didn't get updated, the client will exit because the file it got is different from what was initially _approved_ by Chocolatey. 

In this case, the only option we have is to wait for an update or raise an issue with Chocolatey to get this resolved. We only use Chocolatey as a dependency but cannot actually fix or change anything on their part. 
/close
